testimonials/serializers.py

```diff
@@ -0,0 +1,36 @@
+"""Serializers for testimonials."""
+
+from rest_framework import serializers
+
+from testimonials.models import Attestation
+
+
+class AttestationSerializer(serializers.ModelSerializer):
+    """Serializer for attestations."""
+
+    avatar = serializers.SerializerMethodField()
+    avatar_small = serializers.SerializerMethodField()
+    avatar_medium = serializers.SerializerMethodField()
```

Comment:
we should be able to remove the SerialerMethodField and replace with the following:
```
    avatar = serializers.URLField(source='avatar.url')
    avatar_small = serializers.URLField(source='avatar_small.url')
    avatar_medium = serializers.URLField(source='avatar_medium.url')
    cover = serializers.URLField(source='cover.url')
```